### PR TITLE
lodash version update

### PR DESCRIPTION
**Pull request by hasantayyar**
There is a security issue with current lodash.

more info https://nodesecurity.io/advisories\?search\=lodash\&version\=3.0.1\#

### CODE REVIEW

**Comment by hasantayyar** [comment]
Also #2 contains an update of lodash version but it is still not enough

```
lodash
Vulnerable:<4.17.5
Patched:>=4.17.5
```

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "email" : "me@daniel.gs"
   },
   "dependencies": {
-    "lodash": "3.0.x"
+    "lodash": "4.17.5"
```
Comment (wojtekmaj) [inline package.json]: `^` maybe?

Comment (hasantayyar) [reply package.json]: absolutely!

Comment (hasantayyar) [reply package.json]: updated.

**Comment by cadesalaberry** [comment]
Can this be merged soon?

**Comment by fcastilloec** [comment]
Any reason why is being delayed for so long? If there's something that needs changing before merging, let us know maybe somebody else can address that so this can get merged.

**Comment by fcastilloec** [comment]
Thanks @d5 , if it's not too much to ask, could you also deploy a new version that would include this PR?

**Comment by d5** [comment]
Sorry for the delay. I wasn't actively maintaining this repo. I just published the new version to npm.

**Pull request merged**